walletlibrary/src/main/java/com/microsoft/did/sdk/VerifiableCredentialSdk.kt

```diff
@@ -67,11 +67,13 @@ internal object VerifiableCredentialSdk {
         logConsumer: SdkLog.Consumer = DefaultLogConsumer(),
         polymorphicJsonSerializers: SerializersModule = Json.serializersModule,
         registrationUrl: String = "",
-        resolverUrl: String = "https://discover.did.msidentity.com/v1.0/identifiers"
+        resolverUrl: String = "https://discover.did.msidentity.com/v1.0/identifiers",
+        walletLibraryVersionInfo: String = ""
```

Comment:
Is this temporary? Should we add a TODO to remember to change how this is implemented once we implement a new HTTP Client layer?

Reply:
Added a TODO.